src/index.js

```diff
@@ -147,6 +147,8 @@ document.addEventListener("DOMContentLoaded", () => {
     const id = Number(task.dataset.id);
     database.toggleDone(id, isDone, () => {
       task.classList.toggle("is-done", isDone);
+      task.classList.toggle("is-primary", !isDone);
+      task.classList.toggle("is-secondary", isDone);
```

Comment:
There is no "is-secondary" color class in bulma. Perhaps you meant to say "is-dark"?

Reply:
Fixed in #36699a6